src/main/java/core/basesyntax/service/impl/FilerReadServiceImpl.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.FileReadService;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+
+public class FilerReadServiceImpl implements FileReadService {
+
+    @Override
+    public List<String> readFromFile(String filePath) {
+        try {
+            List<String> lines = Files.readAllLines(Path.of(filePath));
+            lines.remove(0);
```

Comment:
Two times???
do it in parser.
FileReadService can be used everywhere to read any file. First line removing is unexpected.

![image](https://user-images.githubusercontent.com/20326858/219027056-502ded06-8b29-41d9-8c91-f19d73ab0d8b.png)

Reply:
yes)))Sorry